src/traccuracy/utils.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import copy
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from traccuracy._tracking_graph import NodeFlag, TrackingGraph
+
+if TYPE_CHECKING:
+    from traccuracy.matchers._base import Matched
+
+
+def get_corrected_division_graphs_with_delta(
```

Comment:
Just checking, this is intended as a utility function in case someone wants to export the graph for visualization or some other task with the annotations for a particular frame buffer.